pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.37.0-dev45"
+__version__ = "0.37.0-dev46"
```

Comment:
```suggestion
__version__ = "0.37.0-dev45"
```

Reply:
Ringo bot should auto-handle this in future

Reply:
I think it made a change, but then there was a conflict with the ci and in manually resolving it, I messed it up further.